references/classification/README.md

```diff
@@ -67,3 +67,24 @@ For Mobilenet-v2, the model was trained with quantization aware training, the se
 Training converges at about 10 epochs.
 
 For post training quant, device is set to CPU. For training, the device is set to CUDA
+
+### Command to evaluate quantized models using the pre-trained weights:
+For all quantized models except inception_v3:
+```
+python references/classification/train_quantization.py  --data-path='imagenet_full_size/' \
+    --device='cpu' --test-only --backend='fbgemm' --model='<model_name>'
```

Comment:
I think mobilenet doesn't use `fbgemm` backend, @raghuramank100 can you confirm?